src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/HartenLaxVanLeer.hpp

```diff
@@ -0,0 +1,211 @@
+        const db::item_type<VariablesTags>&... u_to_package,
+        const db::item_type<char_speeds_tag>& characteristic_speeds) noexcept {
+      expand_pack((get<VariablesTags>(*packaged_data) = u_to_package)...);
+      expand_pack(
+          (get<NormalDotFluxTags>(*packaged_data) = n_dot_f_to_package)...);
+
+      get<MinSignalSpeed>(*packaged_data) = make_with_value<Scalar<DataVector>>(
+          characteristic_speeds[0],
+          std::numeric_limits<double>::signaling_NaN());
+      get<MaxSignalSpeed>(*packaged_data) = make_with_value<Scalar<DataVector>>(
+          characteristic_speeds[0],
+          std::numeric_limits<double>::signaling_NaN());
+
+      for (size_t s = 0; s < characteristic_speeds[0].size(); ++s) {
+        // This initial value ensures that local_min_signal_speed <= 0.0
+        double local_min_signal_speed = 0.0;
+        // Likewise, local_max_signal_speed >= 0.0
+        double local_max_signal_speed = 0.0;
+        for (size_t u = 0; u < characteristic_speeds.size(); ++u) {
+          local_min_signal_speed = std::min(
+              local_min_signal_speed, gsl::at(characteristic_speeds, u)[s]);
+          local_max_signal_speed = std::max(
+              local_max_signal_speed, gsl::at(characteristic_speeds, u)[s]);
+        }
+        get(get<MinSignalSpeed>(*packaged_data))[s] = local_min_signal_speed;
+        get(get<MaxSignalSpeed>(*packaged_data))[s] = local_max_signal_speed;
+      }
+    }
+  };
+
+  template <typename NormalDotNumericalFluxTagList, typename VariablesTagList,
+            typename NormalDotFluxTagList>
+  struct call_operator_helper;
+
+  template <typename... NormalDotNumericalFluxTags, typename... VariablesTags,
+            typename... NormalDotFluxTags>
+  struct call_operator_helper<tmpl::list<NormalDotNumericalFluxTags...>,
+                              tmpl::list<VariablesTags...>,
+                              tmpl::list<NormalDotFluxTags...>> {
+    static void function(
+        const gsl::not_null<
+            db::item_type<NormalDotNumericalFluxTags>*>... n_dot_numerical_f,
+        const db::item_type<NormalDotFluxTags>&... n_dot_f_interior,
+        const db::item_type<VariablesTags>&... u_interior,
+        const db::item_type<MinSignalSpeed>& min_signal_speed_interior,
+        const db::item_type<MaxSignalSpeed>& max_signal_speed_interior,
+        const db::item_type<NormalDotFluxTags>&... minus_n_dot_f_exterior,
+        const db::item_type<VariablesTags>&... u_exterior,
+        const db::item_type<MinSignalSpeed>& min_signal_speed_exterior,
+        const db::item_type<MaxSignalSpeed>&
+            max_signal_speed_exterior) noexcept {
+      auto min_signal_speed = make_with_value<db::item_type<MinSignalSpeed>>(
+          min_signal_speed_interior,
+          std::numeric_limits<double>::signaling_NaN());
+      auto max_signal_speed = make_with_value<db::item_type<MaxSignalSpeed>>(
+          min_signal_speed_interior,
```

Comment:
If this works with `max_signal_speed_interior`, could you use that instead? Just to reduce possible future confusion later as to whether this should be `min...`